pkg/registries/adapters/rhcc_adapter.go

```diff
@@ -56,11 +56,11 @@ func (r RHCCAdapter) FetchSpecs(imageNames []string) ([]*apb.Spec, error) {
 	specs := []*apb.Spec{}
 	for _, imageName := range imageNames {
 		req, err := http.NewRequest("GET",
-			fmt.Sprintf("%v/v2/%v/manifests/latest", r.Config.URL.String(), imageName), nil)
+			fmt.Sprintf("%v/v2/%v/manifests/%v", r.Config.URL.String(), imageName, r.Config.Tag), nil)
```

Comment:
I think there's a `Validate` on the Config to make sure that a value is entered for Tag. If no value is present default it to "latest". Otherwise, I can change the value in the deployment template to empty and we'll be appending `:` to the end of the image names with no tags. Not sure what will happen.